src/app/formalVerification.js

```diff
@@ -0,0 +1,30 @@
+var $ = require('jquery');
+
+function FormalVerification (outputElement, renderer) {
+  this.outputElement = outputElement;
+  this.renderer = renderer;
+}
+
+FormalVerification.prototype.compiling = function () {
+  $('#formalVerificationInput', this.outputElement).val('');
+  $('#formalVerificationErrors').empty();
+};
+
+FormalVerification.prototype.compilationFinished = function (compilationResult) {
+  if (compilationResult.formal !== undefined) {
```

Comment:
Shouldn't it display a message if formal verification is lacking in the solc version used?